### PR TITLE
Header with pepcus core.

### PR DESCRIPTION
**Pull request by BhagyashreeBhoyar**
Generic header with Pepcus core.

### CODE REVIEW

File: src/components/common/Header.jsx [hunk 1]
```diff
@@ -1,19 +1,127 @@
...
+    color: ${getThemeProps('palette.white.color')};
+    text-decoration: none;
+    font-weight: 600;
+    font-size: 16px;
+    padding: 2px 15px;
+    border: 1px solid ${getThemeProps('palette.white.color')};
+    margin-right: 10px;
+    border-radius: 3px;
+    &:hover {
+    background-color: ${getThemeProps('palette.primary.color')};
```
Comment (pepcus-pratik) [inline src/components/common/Header.jsx]: Improper indentation

File: src/components/common/Header.jsx [hunk 2]
```diff
@@ -25,7 +133,7 @@ import { DEFAULT_HEADER_TEXT } from '../../constants/yjsg';
  * @param {Function} resetVisibleColumnConfig
  * @param {String} tenant
  * @type {Function}
- * @return {ReactComponent}
+ * @return {HTML}
```
Comment (pepcus-pratik) [inline src/components/common/Header.jsx]: @constructor never @return

**Comment by pepcus-pratik** [comment]
@Nikmahajan I've reviewed this PR

File: src/components/common/Header.jsx (hunk 1 above)
Comment (BhagyashreeBhoyar) [reply src/components/common/Header.jsx]: Done.

File: src/components/common/Header.jsx (hunk 2 above)
Comment (BhagyashreeBhoyar) [reply src/components/common/Header.jsx]: Done.

**Pull request merged**